css/style.css

```diff
@@ -1,50 +1,275 @@
+  color: #FFFFFF;
+  font-weight: 600;
+  text-decoration: none;
+  font-size: 24px;
+}
+.navigation ul {
+  display: flex;
+  justify-content: flex-end;
+  flex-direction: row;
+  padding: 40px;
+}
 
-/******************************************
-/* ADDITIONAL STYLES
-/*******************************************/
\ No newline at end of file
+.navigation li {
+  padding-right: 40px;
+  list-style: none;
+}
+
+.navigation h1 {
+  font-weight: 800;
+  font-size: 46px;
+  color: #FFFFFF;
+  display: flex;
+  justify-content: flex-start;
+  flex-direction: row;
+}
+
+.content {
+  display: flex;
+  justify-content: space-between;
+  flex-grow: 1;
+  align-items: baseline;
+  font-family: 'Open Sans', sans-serif;
+}
+
+.blog {
+  width: 60%;
+  margin-top: 80px;
+  margin-left: 40px;
+  display: flex;
+  flex-direction: column;
+
+}
+
+.blog article {
+  border-bottom: 1px solid #043049;
+  margin-bottom: 40px;
+}
+
+.blog h2 {
+  font-size: 40px;
+  font-weight: 600;
+  line-height: 1.3;
+  color: #043049;
+  padding-bottom: 30px;
+
+}
+
+.blog img {
+  padding-bottom: 40px;
+  width: 700px;
+  height: auto;
+}
+
+.blog p {
+  color: #616161;
+  font-size: 24px;
+  padding-bottom: 30px;
+}
+
+.dropcap {
+  float: left;
+  color: #616161;
+  font-size: 60px;
+  font-weight: 600;
+  line-height: 60px;
+  padding:8px 8px 5px 0;
+}
+
+.blog a {
+  color: #043049;
+  font-size: 24px;
+  font-weight: 600;
+  text-decoration: none;
+  display: flex;
+  justify-content: flex-end;
+  padding-bottom: 30px;
+}
+
+
+.sidebar {
+  width: 40%;
+  display: inline-block;
+  margin-top: 80px;
+  margin-left: 40px;
+  vertical-align: top;
+  padding-left: 20px;
+  font-family: 'Open Sans', sans-serif;
+}
+
+
+.sidebar a {
+  color: #043049;
+  font-size: 22px;
+  font-weight: 600;
+  text-decoration: none;
+}
+
+.sidebar h2 {
+  font-size: 32px;
+  color: #043049;
+  font-weight: 800;
+  padding-bottom: 10px;
+}
+
+.sidebar li {
+  font-size: 24px;
+  font-weight: 200;
+  margin: 0;
+  padding: 0;
+  color: #043049;
+  list-style-type: none;
+
+}
+
+.sidebar ul {
+  padding-bottom: 50px;
+}
+
+.sidebar p {
+  font-size: 22px;
+  font-weight: 200;
+  line-height: 1.2;
+  color: #043049;
+}
+
+.sidebar img {
+  padding-top: 50px;
+}
+
+
+
+
+
+#ctabox {
+  background: #f9e42e;
+  padding: 50px;
+  text-align: center
+}
+
+.button {
+  border-radius: 5px;
+  font-weight: 800;
+  padding: 12px 0;
+  width: 280px;
+  font-size: 28px;
+  text-align: center;
+  display: block;
+  margin: 0 auto;
+}
+
+
+.blue {
+  background: #033048;
+  color: #fff;
+  font-size: 24px;
+}
+
+footer {
+  background: #121212;
+  padding: 65px 0 45px;
+  text-align: center;
+}
+
+footer .logo {
+  color: #f9e42e;
+  margin-bottom: 68px;
+}
+
+footer ul li {
+  display: inline-block;
+  list-style: none;
+  width: 50px;
+  position: relative;
+  margin: 0 26px 75px;
+}
+
+
+footer ul {
+  display: flex;
+  max-width: 200px;
+  margin: auto;
+  padding: 0;
+  justify-content: space-between;
```

Comment:
if you set the element to `display: inline-flex;` you get all the benefits of `display: flex` but it won't act like a block element. This means that you can then remove `max-width` and the `justify-content`